server/models/Message.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const messageSchema = mongoose.Schema({
+  conversationId: {
+    type: Schema.Types.ObjectId,
+    ref: "Conversation",
+    required: true,
+  },
+  senderId: {
+    type: Schema.Types.ObjectId,
+    ref: "User",
+    required: true,
+  },
+  messageText: {
+    type: String,
+    required: true,
+  },
+  messageStatus: {
+    type: Boolean,
+    default: false,
+    required: true,
+  },
+  createdAt: {
```

Comment:
Just a suggestion - for future you can consider using mongoose timestamps which will generate this field automatically

Ref: https://masteringjs.io/tutorials/mongoose/timestamps